src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js

```diff
@@ -152,18 +148,17 @@ const FormFieldWerkingsgebiedKoppeling = ({
 }
 
 /**
- * Function that renders the CardSelectedWerkingsgebied component, which is used within the FormFieldWerkingsgebiedKoppeling component, while using the imported Transition component for the animation.
- * On this component the user sees a title with the selected werkingsgebied, when the werkingsgebied was last updated,
- * a button with the text "Dit werkingsgebied ontkoppelen" to disconnect the werkingsgebied and an backgroundimage containing a map of the Netherlands that is filtered on the UUID of the
+ * Function that renders the CardSelectedWerkingsgebied component in which the user sees when the selected werkingsgebied is last updated and a option to disconnect a werkingsgebied.
+ *
  *
  * @function
  *
- * @param {function} setPopupOpen - Parameter that is used set the state of the popupOpen variable to true.
- * @param {function} setWerkingsgebiedInParentState - Parameter that is used to set the state value of the werkingsgebiedInParentState variable.
- * @param {function} setWerkingsgebied - Parameter that is used to set the state value of the werkingsgebied variable.
- * @param {object} dataObjectProperty - Parameter that is used to set the value within the target state value of the werkingsgebiedInParentState variable.
- * @param {object} werkingsgebied - Parameter used for conditional operators within the CardSelectedWerkingsgebied component, to display the title, a certain date format and the UUID for the url backgroundimage if it contains a value.
- * @param {boolean} show - Parameter to toggle the Transition component, that displays the CardSelectedWerkingsgebied component in a certain animation.
+ * @param {function} setPopupOpen - Parameter that is used set the state of the popupOpen variable.
+ * @param {function} setWerkingsgebiedInParentState - Function to edit parent state.
+ * @param {function} setWerkingsgebied - Function to edit parent state
+ * @param {object} dataObjectProperty - Parameter containing the werkingsgebied data property.
```

Comment:
typeof string

Reply:
I've changed the typeOf from object ==> string for the dataObjectProperty parameter.